### PR TITLE
Package name encoding in conflict message

### LINKED ISSUE #6030: spurious percent-encoding in output
Hi,
I'm using opam 2.2.0~beta3 on windows. I've noticed that a package name is percent-encoded in an error message:

```
C:\Users\etien\src\dune>opam install -y lwt cinaps core_bench "csexp>=1.3.0" js_of_ocaml js_of_ocaml-compiler "mdx>=2.3.0" menhir ocamlfind ocamlformat.0.26.1 "odoc>=2.4.0" "ppx_expect>=v0.16.0" ppx_inline_test
 ppxlib ctypes "utop>=2.6.0" "melange>=4.0.0-414"
[ERROR] Package conflict!
  * No agreement on the version of ppx%5fexpect:
    - ppx_expect >= v0.16.0
    - core_bench → ppx_jane < v0.15 → ppx_expect < v0.15
  * No agreement on the version of ocaml-base-compiler:
    - (invariant) → ocaml-base-compiler = 4.14.1
    - core_bench → core = 109.24.00 → ocaml < 4.01.0 → ocaml-base-compiler = 3.09.1
```

The first occurrence is displayed as `ppx%5fexpect`, while the rest is displayed correctly as `ppx_expect`.
Maybe that's a styling issue, since the invalid one is printed in bold while the other ones are using normal text (this on windows Terminal).

image

### PR DESCRIPTION
**Pull request by rjbou**
fix #6030

### CODE REVIEW

File: master_changes.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ users)
 ## Actions
 
 ## Install
+  * Fix package name display for  no agreement conflicts [#XXX @rjbou - fix #6030]
```
Comment (dra27) [inline master_changes.md:26]:
```suggestion
  * Fix package name display for  no agreement conflicts [#6055 @rjbou - fix #6030]
```

**Review by dra27 — APPROVED** [review]
LGTM - given that it's been wrong for nearly 3 years, I guess we can leave this as a post 2.2.0 fix?

**Comment by rjbou** [comment]
yes, it's not meant to be in 2.2 final

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**